README.md

```diff
@@ -9,7 +9,7 @@ LexikJWTAuthenticationBundle
 
 This bundle provides JWT (Json Web Token) authentication for your Symfony API.
 
-It is compatible and tested with PHP 5.5, 7.2, 7.3, on Symfony 3.4 and 4.2.
+It is compatible and tested with PHP 5.5, 7.2, 7.3, on Symfony 3.4, 4 and 5.
```

Comment:
@abdounikarim Did you verified, it will work for Symfony 4 and symfony5. I have tried, it is not worked for symfony4.3.

Reply:
Yes its verified by tests: https://travis-ci.org/lexik/LexikJWTAuthenticationBundle